services/ui-src/src/components/statusing/TableRow.test.tsx

```diff
@@ -31,7 +31,9 @@ const tableRowComponent = ({ page, rowDepth }: RowProps) => {
   return (
     <RouterWrappedComponent>
       <Table>
-        <TableRow page={page} rowDepth={rowDepth} />
+        <Tbody>
```

Comment:
Fixes `Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.`.